acceptance_tests/features/environment.py

```diff
@@ -115,6 +127,16 @@ def after_scenario(_, scenario):
 def after_step(context, step):
     if step.status == "failed":
         logger.exception('Failed step', scenario=context.scenario.name, step=step.name)
+        if os.getenv('SCREENSHOT', 'False') == 'True':
+            dir_name = get_screenshot_directory()
+            step_str = step.name
+            file_name = os.path.join(dir_name, step_str + '_failed.png')
+            browser.screenshot(file_name)
+
+
+def get_screenshot_directory():
+    acceptance_tests_directory = os.path.join('..', '..')
```

Comment:
This is probably unnecessary given that we run exclusively in Posix environments, but the use of `os.path.join` proofs us against issues with running on operating systems that don't use `/` as a path separator.